client-js/app/elements/labrad-nodes.ts

```diff
@@ -331,152 +392,227 @@ export class LabradNodes extends polymer.Base {
   }
 
 
-  /**
-   * Find the index in the node info list of a node with the given name.
-   *
-   * If no such node is found, returns -1.
```

Comment:
I figured it was self evident given this is how `indexOf` works, but I can put it back.